tools/fuchsia/gen_package.py

```diff
@@ -91,20 +91,12 @@ def main():
   ]
 
   # Build the package
-  try:
-    output = subprocess.check_output(pm_command_base + ['build'])
-  except subprocess.CalledProcessError as e:
-    print('The "%s" command failed:' % e.cmd)
-    print(e.output)
-    raise
+  # Use check_output so if anything goes wrong we get the output.
+  output = subprocess.check_output(pm_command_base + ['build'])
```

Comment:
do we need to do something with this output? like logging it or does `check_output` pipe `stderr` in a way that LUCI can capture it?

Reply:
If the exception is raised, the exception object will contain the output.  It doesn't do that for `check_call`.

Actually, I think we don't even need to store the output - I was thinking that would prevent printing it but that doesn't make sense.